layers/+email/mu4e/packages.el

```diff
@@ -36,6 +38,9 @@
       (add-to-list 'mu4e-view-actions
                    '("View in browser" . mu4e-action-view-in-browser) t)
 
+      (when (configuration-layer/layer-usedp 'org)
+        (require 'org-mu4e))
```

Comment:
Consider the 2 cases where only 1 of the packages is loaded.

Only mu4e (e.g., viewing email): org layer is enable but not loaded, thus `(with-eval-after-load 'org..` will not have been run, and so the org-mu4e functionality will not be loaded. This is why we `(require 'org-mu4e)` if the layer is enabled -- and yes it will cause org to be loaded at that point, but that's ok.

Only org loaded (e.g., looking at TODO file): goto link for mail message it will not open unless org-mu4e has been loaded, this is why we have the `(with-eval-after-load 'org...`